astroquery/svo_fps/core.py

```diff
@@ -9,11 +9,23 @@
 from . import conf
 
 from ..query import BaseQuery
+from astroquery.exceptions import InvalidQueryError
+
 
 __all__ = ['SvoFpsClass', 'SvoFps']
 
 FLOAT_MAX = np.finfo(np.float64).max
 
+# Valid query parameters taken from
+# http://svo2.cab.inta-csic.es/theory/fps/index.php?mode=voservice
```

Comment:
Is there any way to parse this in rather than hard wire the params? I suppose they may not change it too often, so the hard-wiring should also work in practice.

Reply:
If the parameter names are not hard-coded here then they have to be monkeypatched for non-remote tests, which requires them to be hard-coded there. I have instead added a new remote test that should raise alarms if `QUERY_PARAMETER` becomes outdated.

Reply:
That's a solid middle ground, thanks.